etc/adminhtml/system.xml

```diff
@@ -42,7 +42,7 @@
             </group>
             <group id="product_protection" translate="label" type="text" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
                 <label>Product Protection</label>
-                <field id="enable" translate="label" type="select" sortOrder="10" showInDefault="1" canRestore="1">
+                <field id="enable" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
```

Comment:
Was this approved higher up?

Reply:
yeah - per the ticket's open questions section:

> Q: Are the Product Protection & Shipping Protection toggles supposed to be visible in the website or store scopes, or should they remain only at the default (global) level?
> A: Store level